### PR TITLE
Completed css-flexbox.

### PR DESCRIPTION
**Pull request by jayb0c**
Screen Shot 2021-06-30 at 9 46 44 PM
Screen Shot 2021-06-30 at 9 30 01 PM
Screen Shot 2021-06-30 at 9 40 22 PM

### CODE REVIEW

File: css-flexbox/display-flex.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - display: flex</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
```
Comment (BrettAlbright) [inline css-flexbox/display-flex.html:11]: you must complete the code for this page.

File: css-flexbox/flex-wrap.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
```
Comment (BrettAlbright) [inline css-flexbox/flex-wrap.html]: this section as well.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Make sure to complete the other two pages

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**